test/gen_prog_align.jl

```diff
@@ -0,0 +1,14 @@
+@testset "GenProgAlign" begin
+using STMOZOO.GenProgAlign
+    @testset "ga_alignment" begin
```

Comment:
You have a lot of code here and few tests, is it possible to add more? For example, test some edge cases or error handing.